### PR TITLE
[WIP] Actioncable notifications

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

**Comment by mjgiarlo** [comment]
oooooooooooooh

File: config/cable.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 development:
-  adapter: async
+  # adapter: async
+  adapter: redis
+  url: redis://localhost:6379/1
```
Comment (mjgiarlo) [inline config/cable.yml]: Could we use ENV var jazz in this new file so we're cool in AWSland?

File: app/channels/notifications_channel.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class NotificationsChannel < ApplicationCable::Channel
+  def subscribed
+    stream_from "notifications:1"
```
Comment (mjgiarlo) [inline app/channels/notifications_channel.rb]: Is the `1` here, and down in the Redis URL below, a Redis "database" to keep these keys cordoned off from other ones?

**Comment by atz** [comment]
@jcoyne still WIP?  should we rebase or kill this?

**Comment by jcoyne** [comment]
Yep, it's still a thing. Needs rebase and tests

**Comment by mjgiarlo** [comment]
I hope this can be superseded by https://github.com/samvera/hyrax/pull/1624

Though we will likely need to do some work in Hyku to make sure the above impl works as expected in a multi-tenant instance.

**Comment by mjgiarlo** [comment]
Closing per above comment.

**Pull request closed without merging**